site2/docs/schema-manage.md

```diff
@@ -809,17 +809,118 @@ To use your custom schema storage implementation, perform the following steps.
 
 ## Set schema compatibility check strategy 
 
-You can set [schema compatibility check strategy](schema-evolution-compatibility.md#schema-compatibility-check-strategy) at namespace or broker level. 
+You can set [schema compatibility check strategy](schema-evolution-compatibility.md#schema-compatibility-check-strategy) at the topic, namespace or broker level. 
 
-- If you set schema compatibility check strategy at both namespace or broker level, it uses the strategy set for the namespace level.
+The schema compatibility check strategy set at different levels has priority: topic level > namespace level > broker level. 
 
-- If you do not set schema compatibility check strategy at both namespace or broker level, it uses the `FULL` strategy.
+- If you set the strategy at both topic and namespace level, it uses the topic-level strategy. 
 
-- If you set schema compatibility check strategy at broker level rather than namespace level, it uses the strategy set for the broker level.
+- If you set the strategy at both namespace and broker level, it uses the namespace-level strategy.
 
-- If you set schema compatibility check strategy at namespace level rather than broker level, it uses the strategy set for the namespace level.
+- If you do not set the strategy at any level, it uses the `FULL` strategy.
 
```

Comment:
Could you add a description that available values for schema compatibility strategy?

Reply:
Good point. I will add a link as the reference for available values. Thank you.